components/scheduler/views/agenda.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Agenda
+page_title: Scheduler - Agenda View
+description: Agenda View in the Scheduler for Blazor.
+slug: scheduler-views-agenda
+tags: telerik,blazor,scheduler,view,agenda
+published: True
+position: 6
+---
+
+# Agenda View
+
+The Agenda view of the Scheduler for Blazor shows a weekly summary (or a custom period set by the user) in a table format. Also the Agenda view can be configured to display the events that are [grouped by resource]({%slug scheduler-resource-grouping%})
+
+
+In this article:
+
+* [View Parameters](#view-parameters)
+* [Example](#example)
+
+## View Parameters
+
+The following parameters allow you to configure the agenda view:
```

Comment:
You need to decide on a consistent spelling for the Agenda view. I recommend going with `Agenda view`, rather than `agenda view`, but it's up to you. Apply as needed throughout your docs.